main.js

```diff
@@ -0,0 +1,36 @@
+const menuBtn = document.querySelector('#menu-btn');
+const menuBtnIcons = menuBtn.querySelectorAll('.menu-btn img');
+const menu = document.querySelector('.nav-list');
+const menuLinks = menu.querySelectorAll('.nav-list a');
+const body = document.querySelector('body');
+
+function showMenu() {
+  body.classList.toggle('overflow');
+  menu.classList.toggle('toggled-menu');
+
+  menuBtnIcons.forEach((icon) => {
+    icon.classList.toggle('hidden');
+  });
+
+  menuLinks.forEach((link) => {
+    link.classList.toggle('nav-link');
+  });
+}
```

Comment:
[OPTIONAL] It would be great to use the ES6 arrow function syntax in order to follow javascript best practices. Using this method keeps your code clean and is also more readable 👍